otelcol-config.yml

```diff
@@ -2,13 +2,15 @@ receivers:
   otlp:
     protocols:
       grpc:
+        endpoint: 0.0.0.0:4317
       http:
+        endpoint: 0.0.0.0:4318
 exporters:
-  jaeger:
-    endpoint: "jaeger:14250"
+  otlp:
+    endpoint: "jaeger:4317"
```

Comment:
Should 'jaeger' also be modified here?

Reply:
Oh hm, disregard i suppose? Do we need to expose this port on the jaeger container tho? Sorry been a minute since I've peaked at this setup

Reply:
Don't think it's necessary to expose the port on the jaeger container, as the data flows through the collector to jaeger.